Jenkinsfile

```diff
@@ -1,33 +1,5 @@
-@Library('fxtest@1.6') _
+@Library('fxtest@servicebook') _
 
-pipeline {
-  agent any
-  options {
-    ansiColor('xterm')
-    timestamps()
-    timeout(time: 5, unit: 'MINUTES')
-  }
-  environment {
-    PYTEST_ADDOPTS = "-n=10 --color=yes"
-  }
-  stages {
-    stage('Test') {
-      steps {
-        sh "${WORKSPACE}/run"
-      }
-    }
-  }
-  post {
-    failure {
-      mail(
-        body: "${BUILD_URL}",
-        from: "firefox-test-engineering@mozilla.com",
-        replyTo: "firefox-test-engineering@mozilla.com",
-        subject: "Build failed in Jenkins: ${JOB_NAME} #${BUILD_NUMBER}",
-        to: "chartjes@mozilla.com")
-    }
-    changed {
-      ircNotification('#services-test')
-    }
-  }
-}
+
+def sb = new org.mozilla.fxtest.ServiceBook()
+sb.testProject('kinto')
```

Comment:
Note that if we want notifications, we'll need to add these to the `testProject` method, perhaps triggered by arguments. Ultimately it would be nice to switch to declarative, which would require [JENKINS-42224](https://issues.jenkins-ci.org/browse/JENKINS-42224).